### PR TITLE
Refactored to fit PHP 7.4+

### PR DESCRIPTION
**Pull request by bckp**

### CODE REVIEW

**Comment by bckp** [comment]
To discussion is, should we pass prev exception? I belive so, but so far it is not there...

File: src/Console/Command/AbstractConsumerCommand.php [hunk 1]
```diff
@@ -35,11 +35,12 @@ protected function validateConsumerName(string $consumerName): void
 
 		} catch (ConsumerFactoryException $e) {
 			throw new \InvalidArgumentException(
-				$e->getMessage() . sprintf(
-					"\n\n Available consumers: %s",
-					implode('', array_map(static function($s): string {
-						return "\n\t- [{$s}]";
-					}, $this->consumersDataBag->getDataKeys()))
+				sprintf(
+					"{$e->getMessage()}\n\n Available consumers: %s",
```
Comment (paveljanda) [inline src/Console/Command/AbstractConsumerCommand.php:39]:
```suggestion
					"%s\n\n Available consumers: %s",
					$e->getMessage()
```

**Review by paveljanda — CHANGES_REQUESTED** [review]
This way, we use `sprintf` for all string replacements 👍

**Comment by bckp** [comment]
fixed, @paveljanda

**Review by paveljanda — APPROVED** [review]

**Comment by paveljanda** [comment]
Thanks

**Pull request merged**